patches/0062-Subject-PATCH-HFCL-ION4-Target-addition-patch.patch

```diff
@@ -0,0 +1,396 @@
+index ad0fd62..ee12772 100644
+--- a/package/boot/uboot-envtools/files/ipq40xx
++++ b/package/boot/uboot-envtools/files/ipq40xx
+@@ -47,6 +47,10 @@ linksys,ea8300)
+ zyxel,nbg6617)
+ 	ubootenv_add_uci_config "/dev/mtd6" "0x0" "0x10000" "0x10000"
+ 	;;
++hfcl,ion4)
++    env=$(ubootenv_mtdinfo)
++    ubootenv_add_uci_config $env
++    ;;
+ esac
+ 
+ config_load ubootenv
+diff --git a/package/firmware/ipq-wifi/Makefile b/package/firmware/ipq-wifi/Makefile
+index 05f4cd9..9f48030 100644
+--- a/package/firmware/ipq-wifi/Makefile
++++ b/package/firmware/ipq-wifi/Makefile
+@@ -39,7 +39,8 @@ ALLWIFIBOARDS:= \
+ 	qxwlan_e2600ac \
+ 	tp-link_ap2220 \
+ 	tp-link_ec420-g1 \
+-	cig_wf610d
++	cig_wf610d	\
++	hfcl_ion4
+ 
+ ALLWIFIPACKAGES:=$(foreach BOARD,$(ALLWIFIBOARDS),ipq-wifi-$(BOARD))
+ 
+@@ -118,5 +119,7 @@ $(eval $(call generate-ipq-wifi-package,qxwlan_e2600ac,Qxwlan E2600AC))
+ $(eval $(call generate-ipq-wifi-package,tp-link_ap2220,tp-link AP2220))
+ $(eval $(call generate-ipq-wifi-package,tp-link_ec420-g1,tp-link EC420-G1))
+ $(eval $(call generate-ipq-wifi-package,cig_wf610d,CIG WF_610D))
++$(eval $(call generate-ipq-wifi-package,hfcl_ion4,HFCL ION4))
++
+ 
+ $(foreach PACKAGE,$(ALLWIFIPACKAGES),$(eval $(call BuildPackage,$(PACKAGE))))
+diff --git a/target/linux/ipq40xx/base-files/etc/board.d/01_leds b/target/linux/ipq40xx/base-files/etc/board.d/01_leds
+index 0336617..858e118 100755
+--- a/target/linux/ipq40xx/base-files/etc/board.d/01_leds
++++ b/target/linux/ipq40xx/base-files/etc/board.d/01_leds
+@@ -68,6 +68,10 @@ cig,wf610d)
+ 	ucidef_set_led_wlan "wifi2g" "WIFI2G" "wf6203:green:wifi2g" "phy0tpt"
+ 	ucidef_set_led_wlan "wifi5g" "WIFI5G" "wf6203:green:wifi5g" "phy1tpt"
+ 	;;
++hfcl,ion4)
++    ucidef_set_led_wlan "wlan2g" "WLAN2G" "yellow:wlan2g" "phy0tpt"
++    ucidef_set_led_wlan "wlan5g" "WLAN5G" "red:wlan5g" "phy1tpt"
++    ;;
+ 
+ *)
+ 	;;
+diff --git a/target/linux/ipq40xx/base-files/etc/board.d/02_network b/target/linux/ipq40xx/base-files/etc/board.d/02_network
+index bd28bcf..97a4825 100755
+--- a/target/linux/ipq40xx/base-files/etc/board.d/02_network
++++ b/target/linux/ipq40xx/base-files/etc/board.d/02_network
+@@ -51,6 +51,7 @@ ipq40xx_setup_interfaces()
+ 		ucidef_set_interface_lan "eth0"
+ 		;;
+ 	tp-link,ec420-g1|\
++    hfcl,ion4|\
+ 	cig,wf610d)
+ 		ucidef_set_interface_wan "eth0"
+ 		ucidef_set_interface_lan ""
+diff --git a/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata b/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
+index 38f248f..57ddc8a 100644
+--- a/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
++++ b/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
+@@ -202,6 +202,9 @@ case "$FIRMWARE" in
+ 		ath10kcal_extract "0:ART" 4096 12064
+ 		ath10kcal_patch_mac_crc $(macaddr_add "$(cat /sys/class/net/eth0/address)" 9) #2.4G, wlan1
+ 		;;
++    hfcl,ion4)
++        ath10kcal_extract "0:ART" 4096 12064
```

Comment:
bad indenting

Reply:
This is due tab space we will resolve this.